### PR TITLE
New role definitions

### PR DESCRIPTION
**Pull request by nagydan1**
Team retreat workshop

### CODE REVIEW

File: marketing-pr.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
-# PR, Marketing
-####Purpose: Unspecified.
-Accountibility: Copywriting, press outreach, website, social media, blog.
+# Marketing-PR
+
+####Purpose
+- Having continuous good reputation and reach
+
+####Accountibilities
```
Comment (peterlangmar) [inline marketing-pr.md]: overlap with sales

File: marketing-pr.md [hunk 2]
```diff
@@ -9,17 +26,32 @@ Accountibility: Copywriting, press outreach, website, social media, blog.
 ###Facilitator - 
 ###Secretary - 
 
-### Events & community - Aniko
-####Purpose:
+### Events & community - Anikó
+
+####Purpose
 - Generating sales leads
```
Comment (peterlangmar) [inline marketing-pr.md]: hugely overlaps with sales

File: marketing-pr.md (hunk 1 above)
Comment (EggDice) [inline marketing-pr.md]: I think it is alright to have the same purpose but different accountability

Comment (cassus) [inline marketing-pr.md]: I disagree but it's better than it was before and it's good enough for now.

All actions of a role strive to realize the purpose of the role. Thus if two roles have the same purpose their actions should be quite similar.

**Pull request merged**